### PR TITLE
Add to cart

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-3-13
chrome-capture-2022-3-13 (1)
Files to review ---  
                                           [feat: add to cart](https://github.com/aniketsingh13/PlayX-ecom/commit/e686966315905bcb548e64ba1b1eb55479546cb0)
[feat: add to wishList and removed from wishList](https://github.com/aniketsingh13/PlayX-ecom/commit/d9d4eb1d1eb29c724776e01312c64e91c282f13e)
[feat: increase,decrease qty,move to wishlist,remove from cart](https://github.com/aniketsingh13/PlayX-ecom/commit/5ce644911a210dfdc59f2160e5de22d7bad7a86b)

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
all functionality is working, code looks good, good use of helper function  for good readability , good to go

File: play-app/src/Components/Cartcard/Cartcard.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import "./Cartcard.css"
+import {GoPlus} from "react-icons/go";
+import {BiMinus} from "react-icons/bi";
+import { useCart } from "../../Context/CartCont";
+import axios from "axios";
+import { itemWishlist } from "../../Utils/CartUtils/itemWishlist";
```
Comment (TheAmlanSaikia) [inline play-app/src/Components/Cartcard/Cartcard.jsx:6]: In useCart and itemWishlist import you can index.js for each folder

File: play-app/src/Components/Cartcard/Cartcard.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+         } catch (error) {
+             console.log(error.cartResponse)
+         }
+     }
+   }
+
+   const moveToWishlist=(product)=>{
+    cartDispatch({type: 'remove-from-cart',payload: product._id})
+    if(!itemInYourWishlist){
+    cartDispatch({type: "add-to-wishlist", payload: product})
```
Comment (TheAmlanSaikia) [inline play-app/src/Components/Cartcard/Cartcard.jsx:31]: Suggestion: types should be in full caps so to decrease error chances.

Comment (TheAmlanSaikia) [inline play-app/src/Components/Cartcard/Cartcard.jsx:31]: Suggestion: types should be in full caps so to decrease error chances.

File: play-app/src/Components/Cartcard/Cartcard.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+   }
+    
+   const increaseHandler=(_id)=>{
+    cartDispatch({type: "increase-quantity",payload: _id})
+   }
+   
+   const decreaseHandler=(_id)=>{
+       if(qty>1){
+        cartDispatch({type: "decrease-quantity",payload: _id})
+       }
```
Comment (TheAmlanSaikia) [inline play-app/src/Components/Cartcard/Cartcard.jsx:42]: Same as above.

Comment (TheAmlanSaikia) [inline play-app/src/Components/Cartcard/Cartcard.jsx:42]: Same as above.

**Review by TheAmlanSaikia — APPROVED** [review]
Good and clean code. Good UI. The code is well-formatted. Just a left a few suggestions.

File: play-app/src/Components/Cartcard/Cartcard.jsx (hunk 1 above)
Comment (aniketsingh13) [reply play-app/src/Components/Cartcard/Cartcard.jsx:6]: will do that

File: play-app/src/Components/Cartcard/Cartcard.jsx (hunk 2 above)
Comment (aniketsingh13) [reply play-app/src/Components/Cartcard/Cartcard.jsx:31]: will do that

Comment (aniketsingh13) [reply play-app/src/Components/Cartcard/Cartcard.jsx:31]: will do that

File: play-app/src/Components/Cartcard/Cartcard.jsx (hunk 3 above)
Comment (aniketsingh13) [reply play-app/src/Components/Cartcard/Cartcard.jsx:42]: will do

Comment (aniketsingh13) [reply play-app/src/Components/Cartcard/Cartcard.jsx:42]: will do

**Pull request merged**